contracts/src/v0.8/functions/dev/v1_0_0/FunctionsBilling.sol

```diff
@@ -39,7 +39,8 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint32 requestTimeoutSeconds; //                 ║ How many seconds it takes before we consider a request to be timed out
     uint72 donFee; //                                ║ Additional flat fee (in Juels of LINK) that will be split between Node Operators. Max value is 2^80 - 1 == 1.2m LINK.
     uint16 maxSupportedRequestDataVersion; // ═══════╝ The highest support request data version supported by the node. All lower versions should also be supported.
-    uint224 fallbackNativePerUnitLink; // ═══════════╸ fallback NATIVE CURRENCY / LINK conversion rate if the data feed is stale
+    uint224 fallbackNativePerUnitLink; // ═══════════╗ Fallback NATIVE CURRENCY / LINK conversion rate if the data feed is stale
+    uint32 minimumEstimateGasPrice; // ══════════════╝ The lowest amount of wei that will be used as the tx.gasprice when estimating the cost to fulfill the request
```

Comment:
This type is too small, only goes to ~4 gwei.

Reply:
I think we could get away with uint40. Or go up to uint64? May be overkill.

Reply:
Changed to uint40 because it packs nicely. This let's us get ~`1,099`gwei

Reply:
I think that's enough.
But does it pack nicely? 224+40 = 264

The PR looks good, I'll take another look tomorrow before approving.

Reply:
>  But does it pack nicely? 224+40 = 264

I moved one of the uint32s down, so it's now:
slot1: 32+32+32+32+72+16+40 = 256
slot2: 224+32 = 256